go.mod

```
@@ -10,6 +10,7 @@ require (
 	github.com/giantswarm/microerror v0.2.1
 	github.com/giantswarm/micrologger v0.3.1
 	github.com/google/go-cmp v0.5.2
+	github.com/imdario/mergo v0.3.11 // indirect
```

```suggestion
        // Use mergo 0.3.11 due to bug in 0.3.9 merging Go structs.
	github.com/imdario/mergo v0.3.11 // indirect
```

What about adding this comment?

---

Good idea, I added it, PTAL.